pkg/drivers/container.go

```diff
@@ -0,0 +1,58 @@
+package drivers
+
+import (
+	"github.com/MontFerret/ferret/pkg/runtime/core"
+)
+
+type Container struct {
+	drivers map[string]Driver
+}
+
+func NewContainer() *Container {
+	return &Container{
+		drivers: map[string]Driver{},
+	}
+}
+
+func (c *Container) HasDriver(name string) bool {
```

Comment:
I suggest to remove `Driver` word from `Container's` methods. The `Container` is in `drivers` package. Thus when we write `Container.Has` we know it means `has driver`

That's how `Container` calls looks in `ferret.go`:
```go
// drivers 
for _, drv := range i.drivers.RegisteredDrivers() { ... }
```
How will it look like after remove:
```go
for _, drv := range i.drivers.Registered() { ... }
```

In my opinion, the second code looks better

Reply:
I wanted to this way originally, but then I saw that ``Namespace`` interface has ``*Function``, so to be consistent I decided to follow the pattern.

Reply:
I think it will be consistent because `namespace` and `function` are different entities. It's ok to write `Namespace.RegisteredFunctions()` and `Drivers.Registered` together